test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -72,13 +72,16 @@ def compute_qdel_time(self):
         return :
               -1 on qdel fail
         """
+        qsel_comm = os.path.join(self.server.client_conf['PBS_EXEC'],
+                                 'bin',
+                                 'qselect')
+        qselect = "`" + qsel_comm + "`"
         command = self.time_command
         command += " -f \"%e\" "
-        command += os.path.join(
-            self.server.client_conf['PBS_EXEC'],
-            'bin',
-            'qdel `qselect`')
-
+        command += os.path.join(self.server.client_conf['PBS_EXEC'],
+                                'bin',
+                                'qdel ')
+        command += qsel_comm
```

Comment:
I would do this as two steps.  First do the qselect and save the output.  Do the qdel on the saved output.  This has the nice benefit of allowing for the case of no jobs.  If there are no jobs, qselect will return nothing.  In the current way of doing the qdel, it will fail and cause the function to return -1.